frontend/tests/pages/page.test.tsx

```diff
@@ -16,15 +16,15 @@ jest.mock("next-intl", () => ({
 
 describe("Home", () => {
   it("renders intro text", () => {
-    render(<Home />);
+    render(Home({ params: { locale: "en" } }));
```

Comment:
this is a bit unfortunate, but seems to be the best way to deal with testing a top level page that takes params from the Next system